src/web/src/app/app.component.html

```diff
@@ -71,3 +81,5 @@ <h1 class="branding__text" *ngIf="!mobileQuery.matches">SvetIT</h1>
 </main>
 
 <div class="drop-down-overlay" [class.force-visible]="showDropDown" (click)="toggleDropDown()"></div>
+<app-progress-spinner #appSpinner>
+</app-progress-spinner>
```

Comment:
Перенеси пожалуйста на одну строку. И якорь тут кажется не нужен.
```suggestion
<app-progress-spinner></app-progress-spinner>
```

Reply:
Готово.